apl/scalar-functions/conversion-functions/tolong.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: tolong
+description: 'This page explains how to use the tolong function in APL.'
+---
+
+Use the `tolong` function to convert various data types to a long (signed 64-bit) integer value. This is helpful when you need to normalize numeric values from different sources into long integer format for mathematical operations, comparisons, or when working with large numeric identifiers.
+
+You typically use `tolong` when working with numeric strings, floating-point numbers, or other types that need to be converted to long integers, especially when dealing with large numbers that exceed the standard integer range.
```

Comment:
toint/tolong are the exact same, there is no distinction between them in bitdepth. I don't think we should document both independently, its like todouble/toreal